wdl/tasks/hotspot1.wdl

```diff
@@ -37,3 +38,41 @@ task runhotspot {
         disks: resources.disks
     }
 }
+
+
+task enumerate_uniquely_mappable_space {
+    input {
+        BowtieIndex bowtieindex
```

Comment:
`bowtie_index` to be consistent with https://github.com/ENCODE-DCC/dnase-seq-pipeline/blob/98a5792859d0f92427d4c37ddc46328a8ebe9f1c/wdl/subworkflows/build_bowtie_index.wdl#L20